recipes-rubygems/rubygems-sqlite3_1.7.2.bb

```diff
@@ -27,6 +27,11 @@ inherit rubygems
 inherit rubygentest
 inherit pkgconfig
 
+do_install:append() {
+    # remove sqlite3 source code
+    rm -rf ${D}${libdir}/ruby/gems/${GEMLIB_VERSION}/gems/${GEM_NAME}-${PV}/ports
```

Comment:
The Gem expects the native libsqlite library already compiled and installed on the target.
```
GEM_INSTALL_FLAGS:append = " \
    --platform=ruby -- \
    --enable-system-libraries \
"
```

I see no benefit to have the SQLite source code (tarball) on the target. The version of SQLite provided in source code by the Gem might differ from the version provided by openembedded-core. To avoid mess-up with different versions of SQLite, I think it's better to not provide the source code by the Gem.